src/main/java/io/github/cottonmc/resources/item/DefaultItems.java

```diff
@@ -0,0 +1,28 @@
+package io.github.cottonmc.resources.item;
+
+import io.github.cottonmc.cotton.Cotton;
+import io.github.cottonmc.cotton.registry.CommonBlocks;
+import io.github.cottonmc.cotton.registry.CommonItems;
+import net.minecraft.block.Block;
+import net.minecraft.block.OreBlock;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+
+public class DefaultItems {
+
+    /**Registers an Item based on its name if it is supported or returns it if it already exists.
+     * @param name The name of the block that shall be registered if the item hasn't already been registered.
+     * @return The Item associated with the given name or null if the name is not supported.
+     */
+    public static Item provide(String name){
```

Comment:
All the cases here will be identical, won't they?

Reply:
not necessarily, since we might want to support stuff like projectiles or food items...